crates/forge/bin/cmd/test/mod.rs

```diff
@@ -116,9 +116,18 @@ pub struct TestArgs {
     #[arg(long)]
     pub max_threads: Option<u64>,
 
+    /// Show test execution progress.
+    #[arg(long)]
+    pub show_progress: bool,
+
     #[command(flatten)]
     filter: FilterArgs,
 
+    /// Re-run recorded test failures from last run.
+    /// If no failure recorded then regular test run is performed.
+    #[arg(long)]
+    pub run_failures: bool,
```

Comment:
Could possibly be shortened to `--rerun`  for brevity

Reply:
yep, can change. Not sure though it would be obvious that only failures are run if renamed to `--rerun`?

Reply:
changed it in https://github.com/foundry-rs/foundry/pull/8338/commits/55f68c937cd6325925dc627adad9171b8d4acef5 description should be enough to make it clear it reruns only failures :+1: